document/src/vespa/document/util/bytebuffer.h

```diff
@@ -93,14 +73,11 @@ class ByteBuffer
     /** @return Returns the index of the current position in the buffer. */
     size_t getPos() const { return _pos; }
 
-    /** @return Returns the limit. */
-    size_t getLimit() const { return _limit; }
-
     /**
      * @return Returns the number of bytes remaining in the buffer - that is,
      *         getLimit()-getPos().
     */
-    size_t getRemaining() const { return _limit-_pos; }
+    size_t getRemaining() const { return _len -_pos; }
```

Comment:
Consider updating comment to reflect new semantics

Reply:
Done